pygluu/kubernetes/templates/helm/gluu/values.yaml

```diff
@@ -1132,7 +1132,7 @@ oxauth:
     limits:
       # -- CPU limit.
       cpu: 2500m
-      # -- Memory limit.
+      # -- Memory limit. This value is used to calculate memory allocation for Java. Currently it only supports `Mi`. Please refrain from using other units.
```

Comment:
The comments should be reflected to `README.md`. Use `helm docs` to autogenerate the docs.